echo/server/.env.sample

```
@@ -1,94 +1,94 @@
+REDIS_URL=redis://redis:6379
+API_BASE_URL=http://localhost:8000
+ADMIN_BASE_URL=http://localhost:5173
+PARTICIPANT_BASE_URL=http://localhost:5174
+OPENAI_API_BASE_URL=https://api.openai.com/v1
+
+############################################################
+# LiteLLM Model Configurations
+############################################################
+
+# Small Model
+SMALL_LITELLM_MODEL="gpt-4o-mini"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)